docker/ubuntu/bionic/Dockerfile

```diff
@@ -0,0 +1,132 @@
+# Docker build environment for Boost on Ubuntu Bionic
+#
+# Copyright (c) 2018 James E. King III
+#
+# Use, modification, and distribution are subject to the
+# Boost Software License, Version 1.0. (See accompanying file
+# LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+FROM ubuntu:bionic
+MAINTAINER James E. King III <jking@apache.org>
+ENV CONTAINER_USER=boost
+ENV DEBIAN_FRONTEND noninteractive
+WORKDIR /boost
+
+# Make sure we're current
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
```

Comment:
We shouldn't be doing a dist-upgrade....the FROM ubuntu:bionic means that it should be an up-to-date image.

Reply:
Thanks, I can remove that.  I don't think it is harmful, since it will catch ubuntu patches that haven't made it into the upstream docker image, but we don't really need it.